src/Navigation/RequireConditions.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @Author: Erik Slovak <erik.slovak@technologystudio.sk>
+ * @Date: 2022-11-08T08:11:68+01:00
+ * @Copyright: Technology Studio
+**/
+
+import { conditionalNavigationManager } from '../Api/ConditionalNavigationManager'
+import {
+  onResolveConditionsResultAction,
+} from '../Api/NavigationUtils'
+import type {
+  OnActionAttributes,
+  RequireConditionsNavigationAction,
+} from '../Model/Types'
+
+export const onRequireConditionsAction = ({
+  action,
+  getContext,
+  getState,
+  nextOnAction,
+  restArgs,
+}: OnActionAttributes<RequireConditionsNavigationAction>): boolean => {
+  const { conditionList } = action
+  const state = getState()
```

Comment:
you are using getState here, but in onNavigateAction you are using getRootState which one is valid?